sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/SymbolicExecution/CSharp/CSharpExplodedGraph.cs

```diff
@@ -555,6 +555,21 @@ private ProgramState VisitDeclarationExpression(DeclarationExpressionSyntaxWrapp
         {
             // DeclarationExpression is not yet fully supported: https://github.com/SonarSource/sonar-dotnet/issues/2936
 
+            if (ParenthesizedVariableDesignationSyntaxWrapper.IsInstance(wrapper.Designation))
+            {
+                var designation = (ParenthesizedVariableDesignationSyntaxWrapper)wrapper.Designation;
+
+                foreach (var variable in designation.Variables)
```

Comment:
I would add a comment with `<cref>` to point to `VisitSimpleAssignment` - saying that we don't push to the stack because we don't consume them later on

Reply:
I've added a comment but I didn't used `<cref>` since it's meant for use at the method level and the comment makes sense only for this particular case.